### DISCUSSION TITLE
请问能否将程序内的所有的文本进行自定义翻译?

### DISCUSSION THREAD

**Discussion by Soys** [spencerwooo/onedrive-vercel-index#346] [Q&A] [answered]
不太清楚是否有这功能。大概就是将所有程序内文本统一在一个文件内，可根据自己的喜好进行翻译。

另外想请问下，目前我部署的突然显示500错误是怎么回事？

(*新增：在測試的過程中，Ctrl+K搜索似乎會與Chrome的Ctrl+K衝突。)

**Comment by myl7** [comment]
应该是指 i18n？
感觉短时间内不太可能能立刻进行，但长期上可以先等等

**Comment by spencerwooo** [comment]
> 大概就是将所有程序内文本统一在一个文件内，可根据自己的喜好进行翻译。

就那么两个单词……我觉得再加 i18n 维护起来很麻烦，就不太想加。（文档都没有人帮忙翻译维护，我自己精力有限。）

> 另外想请问下，目前我部署的突然显示500错误是怎么回事？

我没看到你公开的 forked 仓库，没办法判断你配置文件具体问题，你一个单独的 500 错误我也无从下手，请提供一些更丰富的错误信息。

**Reply by Soys** [reply]
500错误是本身好好的，也放有3个月了……这几天突然就500错误了。部署了两个，两个都这样子
![ERROR](https://user-images.githubusercontent.com/8133994/152178262-570b2c2f-79dc-4e9e-b099-89b1bb8bc70a.png)

**Reply by spencerwooo** [reply]
你这是好久之前的版本了……尽快更新吧

**Reply by Soys** [reply]
更新的话，是不是本身翻译的这些就要重新翻译了？有什么方法能保留这些翻译吗？

**Reply by spencerwooo** [reply]
> 更新的话，是不是本身翻译的这些就要重新翻译了？有什么方法能保留这些翻译吗？

一方面你需要手动解决冲突因为听你的意思你修改了很多东西，所以你可以试试说不定有些翻译没有冲突可以在新版本更新后保留。

但另一方面新版本跟你现在这个版本的部署方法有很多出入了，你应该还是 2021 版本，2022 版本有用到 Redis 并不需要你自己的 clientId 和 clientSecret。是更新好还是重新部署好看你自己了。

更多情况参考文档吧。

**Reply by Soys** [reply]
> > 更新的话，是不是本身翻译的这些就要重新翻译了？有什么方法能保留这些翻译吗？
> 
> 一方面你需要手动解决冲突因为听你的意思你修改了很多东西，所以你可以试试说不定有些翻译没有冲突可以在新版本更新后保留。
> 
> 但另一方面新版本跟你现在这个版本的部署方法有很多出入了，你应该还是 2021 版本，2022 版本有用到 Redis 并不需要你自己的 clientId 和 clientSecret。是更新好还是重新部署好看你自己了。
> 
> 更多情况参考文档吧。

说来惭愧，我一直以来都是直接点README内的部署按钮，都不太懂如何更新……所以我一直都是部署后手动替换需要翻译的地方，正因如此才会想到i18n的。只需要类似`CodeLoading= ‘Loading file content...’;`这样就好。

**Comment by myl7** [comment, marked as answer]
Ref #359

**Comment by spencerwooo** [comment]
Project is now i18n-ed.
